mail_quoted_reply/models/mail_message.py

```diff
@@ -25,7 +25,7 @@ def _prep_quoted_reply_body(self):
             {body}
             </blockquote>
             </div>
-        """.format(
+        """.format(  # noqa: E501
```

Comment:
isn't the pre-commit config wrong if we need this?

Reply:
Hello, I managed to avoid E501 without adding that tag. Thanks